### DISCUSSION TITLE
link to battery charger on Amazon is broken

### DISCUSSION THREAD

**Discussion by vipercortex** [alanbjohnston/CubeSatSim#275] [Q&A]
The Amazon link for the NiMH battery charger in the BOM does not work anymore:
https://www.amazon.com/gp/product/B07WN2VYWT/

The alternative link does not point at the charger.

Is there an alternative link for the board (either on Amazon or other websites?

**Comment by alanbjohnston** [comment]
I have updated the BOM with this link https://www.digikey.com/en/products/detail/sullins-connector-solutions/PRPC014SACN-RC/2776092
